frontend/components/commons/forms/questions-form/fields/mono-selection/MonoSelectionContainer.component.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <MonoSelectionRenderStyle1Component
+    v-if="styleType === COMPONENT_STYLE.RATING"
+    :options="options"
+    @on-select="onSelect"
+  />
+</template>
+
+<script>
+import { COMPONENT_STYLE } from "./monoSelection.properties";
+
+export default {
+  name: "MonoSelectionContainerComponent",
+  props: {
+    options: {
+      type: Array,
+      required: true,
+    },
+    styleType: {
+      type: String,
+      default: () => COMPONENT_STYLE.RATING,
+    },
+  },
+  model: {
+    prop: "options",
+    event: "on-change",
+  },
+  created() {
+    this.COMPONENT_STYLE = COMPONENT_STYLE;
```

Comment:
not sure to understand this assignment

Reply:
Yes excatly. Without this, `COMPONENT_STYLE` is not accessible in the html template